docs/source/bricks.rst

```diff
@@ -752,6 +781,12 @@ If ``xml_keep_tags=True``, the function returns tag information in addition to t
 
   elements = partition_xml(filename="example-docs/factbook.xml", xml_keep_tags=False)
 
+``partition_xml`` includes a ``max_partition`` parameter that indicates the maximum character length for a document element.
+The default value is ``None``, meaning no maximum character count is applied and
+``partition_xml`` will process the whole XML file as a single element.
```

Comment:
Wait, is it true that it will definitely partition the document as a single element, or is that just a possibility?

Reply:
Yeah you're right, if there are line breaks in there it definitely could get split up into different elements. They might just be weird elements with it being XML and all. Maybe we default that one to 1500 too then?

Reply:
Updated XML to default to 1500 as well.